dbms/tests/queries/0_stateless/00910_client_window_size_detection.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. $CURDIR/../shell_config.sh
+
+function cleanup() { rm -f -- -; }
+trap cleanup EXIT
+
+set -e
+set -o pipefail
+
+${CLICKHOUSE_CLIENT} --help </dev/null | wc -L
+script -e -q -c "${CLICKHOUSE_CLIENT} --help" - </dev/null >/dev/null
```

Comment:
Our team is trying to decrypt this code right now, can you help us?

Reply:
`script` will run command with tty (i.e. `script -eqc sh -c '[ -t 0 ] && echo stdin is tty' < /dev/null`) but (I did not dig into this) for some reason under script it returns `ws_col == 0`, so this verifies that client prints `--help` correctly in this case

Reply:
and `set -e` will verify that `clickhouse-client` exists correctly (and if this is non debug build, which do not have asserts, then AFAICS release build will print new lines endlessly)